README.md

```diff
@@ -616,6 +616,7 @@ API | Description | Auth | HTTPS | CORS |
 | [kanye.rest](https://kanye.rest) | REST API for random Kanye West quotes | No | Yes | Yes |
 | [Medium](https://github.com/Medium/medium-api-docs) | Community of readers and writers offering unique perspectives on ideas | `OAuth` | Yes | Unknown |
 | [Programming Quotes API](https://github.com/skolakoda/programming-quotes-api) | Programming Quotes API for open source projects. | No | Yes | Unknown |
+| [Quote Garden](https://pprathameshmore.github.io/QuoteGarden/) | A REST API for quotes. The database currently includes more than 5000 quotes | No | Yes | Unknown
```

Comment:
Add new API

Reply:
Can you change the description to `REST API for more than 5000 famous quotes`?